setup.sh

```diff
@@ -0,0 +1,214 @@
+#!/bin/bash
+
+ARCH=$(uname -m)
+LATEST_RELEASE_VERSION=$(curl -s "https://api.github.com/repos/paradedb/paradedb/releases/latest" | jq -r .tag_name)
+LATEST_RELEASE_VERSION="${LATEST_RELEASE_VERSION#v}"
+
+set -Eeuo pipefail
+set -e
+
+installDocker() {
+  # Set default values
+  pguser="myuser"
+  pgpass="mypassword"
+  dbname="paradedb"
+
+  echo "Installing docker..."
+
+
+  OPTIONS=("Debian Base" "RHEL Based" "Arch Based")
+
+
+  select opt in "${OPTIONS[@]}"
+  do
+    case $opt in
+      "Debian Base")
+        sudo apt-get install docker -y || false
+        break ;;
+      "RHEL Based")
+        sudo dnf install docker || false
+        break ;;
+      "Arch Based")
+        sudo pacman -Syyu docker || false
+        break ;;
+      *)
+        break ;;
+    esac
+  done
+
+  echo "Successfully Installed Docker✅"
+
+
+  # Prompt for user input
+  read -r -p "Username for Database (default: myuser): " tmp_pguser
+  if [[ -n "$tmp_pguser" ]]; then
+    pguser="$tmp_pguser"
+  fi
+
+  read -r -p "Password for Database (default: mypassword): " tmp_pgpass
+  if [[ -n "$tmp_pgpass" ]]; then
+    pgpass="$tmp_pgpass"
+  fi
+
+  read -r -p "Name for your database (default: paradedb): " tmp_dbname
+  if [[ -n "$tmp_dbname" ]]; then
+    dbname="$tmp_dbname"
+  fi
+
+
+  # Pull Docker image
+  echo "Pulling Docker Image for Parade DB: docker pull paradedb/paradedb"
+  docker pull paradedb/paradedb || { echo "Failed to pull Docker image"; exit 1; }
+  echo "Pulled Successfully ✅"
+
+  # Create Docker container
+  echo "Processing..."
+  docker run \
+    --name paradedb \
+    -e POSTGRES_USER="$pguser" \
+    -e POSTGRES_PASSWORD="$pgpass" \
+    -e POSTGRES_DB="$dbname" \
+    -v paradedb_data:/var/lib/postgresql/data/ \
+    -p 5432:5432 \
+    -d \
+    paradedb/paradedb:latest || { echo "Failed to start Docker container. Please check if an existing container is active or not."; exit 1; }
+  echo "Docker Container started ✅"
+
+  # Provide usage information
+  echo "To use paradedb execute the command: docker exec -it paradedb psql $dbname -U $pguser"
+}
+
+# Please update the debian file with the lastest version here
+installDeb(){
+  echo "Select your distribution"
+
+  echo "Installing dependencies...."
+  # echo "Installing cURL"
+  #
+  # sudo apt-get update || false
+  # sudo apt-get install curl || false
+  #
+  # echo "Successfully Installed cURL✅"
+
+  distros=("bookworm" "jammy" "noble")
```

Comment:
Should probably tell users which is why here. Ubuntu 20,04, 22.04 and 24.04 respectively. I don't know the names top of mind